### PR TITLE
Hero revamp

### PR DESCRIPTION
**Pull request by charlene1804**

### CODE REVIEW

**Comment by georges-gomes** [comment]
@gqio the review link doesn't work for me. Does it work for you?

**Comment by gqio** [comment]
Apparently not,

Checking

**Comment by gqio** [comment]
I made a fix for package selection and released.

But still there is a compilation issue on LCD, works like a charm on Saqara and Bench

I think our support for tailwind throws an exception and I think we should move to vite. Because I don’t want to spend time on this one specially without vite.

File: hero/src/index.vue [hunk 1]
```diff
@@ -360,12 +371,23 @@ export default {
   z-index: -1;
 }
 
-.hover-fix {
+.primary-btn-fix {
   transform: translate(-50%, -50%);
   transition: transform 0.2s;
+  padding: 0;
+  padding-left: 5px;
```
Comment (georges-gomes) [inline hero/src/index.vue:378]: what is this padding left of 5px ?

File: hero/src/index.vue [hunk 2]
```diff
@@ -236,6 +234,19 @@ export default {
   </section>
 </template>
 <style lang="scss" scoped>
+.hero {
+  background-image: #242424;
```
Comment (georges-gomes) [inline hero/src/index.vue:238]: This is fallback color ? Should probably just be transparent and let the background of the app select the color

File: hero/src/index.vue (hunk 1 above)
Comment (charlene1804) [reply hero/src/index.vue:378]: It's a fix so that the play icon svg is visually centered inside my button.

File: hero/src/index.vue (hunk 2 above)
Comment (charlene1804) [reply hero/src/index.vue:238]: Good point thank you I will fix that.

**Pull request closed without merging**